lib/rust/ensogl/core/src/debug/monitor/sampler.rs

```diff
@@ -149,7 +156,7 @@ impl Sampler {
 
 const MB: f64 = (1024 * 1024) as f64;
 
-const DEFAULT_SAMPLER: Sampler = Default::default();
+const DEFAULT_SAMPLER: Sampler = Sampler::default();
```

Comment:
Why this change? Repeating the `Sampler` name does not make any sense. What lint caused it?

Reply:
Default is not a const function, so can't be used here.